README.md

```diff
@@ -1,16 +1,18 @@
-# KeePassXC - KeePass Cross-platform Community Edition
+![logo](https://keepassxc.org/logo.png)
```

Comment:
I think the logo should come below the title initial build status. It's rather large and pushes content down